app/models/Campaign.coffee

```diff
@@ -38,6 +51,12 @@ module.exports = class Campaign extends CocoModel
         levels.push({key: level.get('original'), practice, assessment})
       @levelNumberMap = utils.createLevelNumberMap(levels)
     @levelNumberMap[levelID] ? defaultNumber
+    
+  @levelIsPractice: (level, forceCourseNumbering) ->
+    if forceCourseNumbering
+      return level.practice
+    else
+      return level.practice and / [ABCD]$/.test level.name
```

Comment:
This regex seems pretty fragile.  Can we at minimum put a comment explaining why this will probably work?

Reply:
Moved the comment in the dynamic version here when I integrated the two, added link to commit which provides more explanation.